app/Services/StatsServiceProvider.js

```diff
@@ -0,0 +1,58 @@
+const { LanguagePackage, Group, VocabularyCard } = require('../../database');
+
+async function getNumberOfLanguagePackages(userId) {
+  const number = await LanguagePackage.count({
+    where: {
+      userId,
+    },
+  });
+  return number;
+}
+
+async function getNumberOfActiveGroups(userId) {
+  const number = await Group.count({
+    where: {
+      userId,
+      active: true,
+    },
+  });
+  return number;
+}
+
+async function getNumberOfInactiveGroups(userId) {
+  const number = await Group.count({
+    where: {
+      userId,
+      active: false,
+    },
+  });
+  return number;
+}
+
+async function getNumberOfActiveVocabulary(userId) {
```

Comment:
can we move the active in all functions in this provider into an argument, so we only need 3 functions?